cmd/kindnetd/go.mod

```diff
@@ -3,27 +3,40 @@ module sigs.k8s.io/kind/cmd/kindnetd
 go 1.12
 
 require (
```

Comment:
why is this change so large? have you go mod tidy?

Reply:
err, nope , still catching up with go modules :) let me try to redo it

Reply:
can you take a look if is it right now?